packages/ckeditor5-image/src/imagestyle/converters.js

```diff
@@ -42,19 +42,19 @@ export function modelToViewStyleAttribute( styles ) {
 /**
  * Returns a view-to-model converter converting image CSS classes to a proper value in the model.
  *
- * @param {Array.<module:image/imagestyle/imagestyleediting~ImageStyleFormat>} styles The styles for which the converter is created.
+ * @param {Array.<module:image/imagestyle/imagestyleediting~ImageStyleFormat>} arrangements Arrangements for which the converter is created.
  * @returns {Function} A view-to-model converter.
  */
-export function viewToModelStyleAttribute( styles ) {
-	// Convert only non–default styles.
-	const filteredStyles = styles.filter( style => !style.isDefault );
+export function viewToModelStyleAttribute( arrangements ) {
+	// Convert only non–default arrangements.
+	const nonDefaultArrangements = arrangements.filter( style => !style.isDefault );
```

Comment:
1.  Instead of `modelElement: false` let's have `modelElement: [ 'image', 'imageInline' ]`
2.  Let's normalize all `modelElement` values to array form.
3.  In this converter, let's have two arrays: one for inline arrangements, another for block arrangements.
4.  In L67, let's iterate over arrangements corresponding to the model element name only, thus disallowing loading incorrect arrangements for a particular image type.

Reply:
I named it 'modelElement**s**', since it's an array now.